code/modules/reagents/chemistry/reagents/paradise_pop.dm

```diff
@@ -101,7 +101,7 @@
 									"SECRET TECHNIQUE: TABLE AND DISPOSAL!",
 									// Borers got removed but the below reference stays because its hilarious
 									"[pick("MY BROTHER", " MY DOG", "MY BEST FRIEND", "THE BORER", "GEORGE MELONS", "BADMINS")] DID IT!",
-									"; WHATS SPACE LAW?!",
+									"; WHATS CORP LAW?!",
```

Comment:
I think the previous brain damage line was funnier tbh. Corp law just sounds awkward here

Reply:
Fair enough/